linker/shared/src/main/scala/org/scalajs/linker/checker/IRChecker.scala

```diff
@@ -1211,20 +1193,6 @@ private final class IRChecker(unit: LinkingUnit,
     errorCount += 1
   }
 
-  private def lookupInfo(className: String)(
-      implicit ctx: ErrorContext): Infos.ClassInfo = {
-    inputProvider.loadInfo(className).getOrElse {
-      reportError(s"Cannot find info for class $className")
-      Infos.ClassInfo(className)
-    }
-  }
-
-  private def tryLookupClass(className: String)(
-       implicit ctx: ErrorContext): Either[Infos.ClassInfo, CheckedClass] = {
```

Comment:
After this removal, is the polyfill for `Either.foreach` at the bottom of this file still needed?